device/fw_debug_device.go

```diff
@@ -0,0 +1,45 @@
+package device
+
+import (
+	"fmt"
+)
+
+const (
+	FWDebugPort = uint64(0x402)
```

Comment:
Could you add this link in comment here?
https://github.com/tianocore/edk2/commit/bf23b44d926982dfc9ecc7785cea17e0889a9297

Reply:
done

Reply:
I also removed the field of the struct, because the port is constant for the device.